### PR TITLE
Fixed - Error: Source path does not exist:

### PR DESCRIPTION
**Pull request by nadeemkhan**

### CODE REVIEW

File: config.xml [hunk 1]
```diff
@@ -27,52 +27,52 @@
...
-    <splash src="resources\android\splash\drawable-land-xhdpi-screen.png" density="land-xhdpi"/>
-    <splash src="resources\android\splash\drawable-land-xxhdpi-screen.png" density="land-xxhdpi"/>
-    <splash src="resources\android\splash\drawable-land-xxxhdpi-screen.png" density="land-xxxhdpi"/>
-    <splash src="resources\android\splash\drawable-port-ldpi-screen.png" density="port-ldpi"/>
-    <splash src="resources\android\splash\drawable-port-mdpi-screen.png" density="port-mdpi"/>
-    <splash src="resources\android\splash\drawable-port-hdpi-screen.png" density="port-hdpi"/>
-    <splash src="resources\android\splash\drawable-port-xhdpi-screen.png" density="port-xhdpi"/>
-    <splash src="resources\android\splash\drawable-port-xxhdpi-screen.png" density="port-xxhdpi"/>
-    <splash src="resources\android\splash\drawable-port-xxxhdpi-screen.png" density="port-xxxhdpi"/>
+    <icon src="resources/android/icon/drawable-ldpi-icon.png" density="ldpi"/>
```
Comment (jdnichollsc) [inline config.xml:30]: @nadeemkhan But I understand that this solution only works on Mac, what happen with Windows?

Comment (nadeemkhan) [reply config.xml:30]: i don't think so..

Comment (jdnichollsc) [inline config.xml]: haha me too, maybe we need to remove the resources from the config.xml file...

Comment (jdnichollsc) [reply config.xml:30]: I'm going to accept your pull request using front-slashes https://github.com/driftyco/ionic-cli/issues/1613

Comment (nadeemkhan) [reply config.xml:30]: thanks bro 👍

Comment (jdnichollsc) [reply config.xml:30]: Thanks to you, let me add you to the credits, what is your email? 👍

Comment (nadeemkhan) [reply config.xml:30]: nadeem7khan@gmail.com

Comment (jdnichollsc) [reply config.xml:30]: Done! https://github.com/jdnichollsc/Ionic-ElastiChat-with-Images/blob/master/README.md#collaborators

Comment (nadeemkhan) [reply config.xml:30]: cool 👍

**Pull request merged**